### DISCUSSION TITLE
Javascript function is not defined - Error

### DISCUSSION THREAD

**Discussion by pavangayakwad** [harttle/liquidjs#383] [Q&A]
Using it inside the angular component allowing users to write their own html with javascript functions. 

The template to be rendered.
```
<script type="text/javascript">
function fnDone(v){
alert(v);
}
</script>


{% for d in data %}
<div>
{{d.title}}
<button onclick="fnDone(d.title)">{{d.title}}</button>
</div>
{% endfor %}
```

In the backend, Initialized the engine,
```
_UIengine = new Liquid({
    cache: true
  });
```
...and rendered the template with the data
```
let data = {/* received from API */ };

this._UIengine.parseAndRender(this._template, data)
.then(html => this.assignRenderedHtml(html));
```

The HTML page is coming up nicely however when I click on the button, receiving the fnDone is not defined error.

![image](https://user-images.githubusercontent.com/1834890/131807468-e8846c8d-2aaa-4564-b8a4-c4166b914180.png)

Here is the rendered HTML 
![image](https://user-images.githubusercontent.com/1834890/131807699-9a655e56-f235-459e-80a8-f63ebb3a1508.png)

What am I doing wrong here? Please assist.

**Comment by harttle** [comment]
I believe it should be `onclick="()=>fnDone(d.title)"`
